<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uiMain</class>
 <widget class="QMainWindow" name="uiMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HoshinoAI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="buttonClose">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>75</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>604</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Close"/>
   </widget>
   <widget class="QMenu" name="menu_Other">
    <property name="title">
     <string>&amp;Other</string>
    </property>
    <addaction name="action_Calc"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Other"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="action_Calc">
   <property name="text">
    <string>&amp;Calc</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
